tilt-resources/examples/deployment.yaml

```diff
@@ -0,0 +1,125 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    component: velero
+  name: velero
+  namespace: velero
+spec:
+  selector:
+    matchLabels:
+      deploy: velero
+  strategy: {}
+  template:
+    metadata:
+      annotations:
+        prometheus.io/path: /metrics
+        prometheus.io/port: "8085"
+        prometheus.io/scrape: "true"
+      labels:
+        component: velero
+        deploy: velero
+    spec:
+      containers:
+        - args:
+            - server 
+            - --log-level 
+            - debug       
```

Comment:
This doesn't have to be `debug` by default.

Reply:
No, but I'm opting to offer more for the sake of better discovery, especially for new people onboarding.

Reply:
I think it's better to leave debug enabled. Given that this is a workflow for development, having the debug output is useful to have by default.